Tools/TestWebKitAPI/Tests/WebKitCocoa/ProcessSwapOnNavigation.mm

```diff
@@ -7539,7 +7539,7 @@ static bool hasOverlay(CALayer *layer)
 
     using namespace TestWebKitAPI;
     checkFrameTreesInProcesses(webView.get(), {
-        { "pson://www.webkit.org"_s }, { RemoteFrame }
+        { "pson://www.webkit.org"_s }
```

Comment:
This is now no longer symmetric.  I would hope the RemoteFrame with the file:// frame below would be the same as this.